vmm/src/igvm/igvm_loader.rs

```diff
@@ -369,6 +371,84 @@ pub fn load_igvm(
         }
     }
 
+    #[cfg(feature = "sev_snp")]
+    {
+        use std::time::Instant;
+
+        let mut now = Instant::now();
+
+        memory_manager
```

Comment:
Cannot call `allocate_address_space()` here since it is already it is already invoked in vmm/src/vm.rs.

Reply:
Could you please pint me to the line # of the code?

Reply:
2060